### ISSUE TITLE
Configurable static files dir

### ISSUE THREAD

**Issue by mauricemach** [mauricemach/zappa#6]

**Comment by 525c1e21-bd67-4735-ac99-b4b0e5262290** [comment]
+1 for this

It seems like alot of these issues pertain to exposing express/socket.io to zappa apps rather than adding any new functionality.

Perhaps there's a way to expose everything that express/socket.io/etc can do without requiring DSL 'wrappers' for each facility.

Just my $0.02.

**Comment by mauricemach** [comment]
Yes, certainly access to lower level libs should be facilitated so that we're not left in the dark when there's not an appropriate wrapper at the zappa level. You can currently access the express app at `app().http_server` but that's not a very elegant solution.

That being said, the "functionality" in zappa is mostly the wrapping itself. It's just a way to create express/socket.io apps with less boilerplate!

**Comment by mauricemach** [comment]
Implemented in 0.2.0beta. We now have progressive layers of syntactic sugar to do that:

```
app.use express.static(__dirname + '/public')

use express.static(__dirname + '/public')

use static: __dirname + '/public'

use 'static'
```
